Release r1.1 with 0.1.0-rc.1 version (Fall'25 M3) (#15)
camaraproject/ApplicationEndpointDiscovery: CHANGELOG.md

# Changelog Application Endpoint Discovery

## Table of contents

- **[r1.1 - rc](#r11---rc)**

**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**

The below sections record the changes for each API version in each release as follows:

- for an alpha release, the delta with respect to the previous release
- for the first release-candidate, all changes since the last public release
- for subsequent release-candidate(s), only the delta to the previous release-candidate
- for a public release, the consolidated changes since the previous public release

<!--Repeat the below release section (header 1 and subsections) at the top of this file for each new (pre-)release-->

# r1.1 - rc

## Release Notes

> This release contains the definition and documentation of

Reviewer: ```suggestion
This pre-release contains the definition and documentation of
```

Author: Fixed on commit 4c0f60d

> - application-endpoint-discovery v0.1.0-rc1, a release-candidate pre-release

Reviewer: ```suggestion
- application-endpoint-discovery v0.1.0-rc.1
```
(same information comes later)

The API definition(s) are based on

- Commonalities r3.2
- Identity and Consent Management r3.2

> ## application-endpoint-discovery v0.1.0-rc1

Reviewer: ```suggestion
## application-endpoint-discovery v0.1.0-rc.1
```

> **application-endpoint-discovery v0.1.0-rc1 is the initial new release candidate version of this API, including initial documentation and implementation code.**

Reviewer: ```suggestion
**application-endpoint-discovery v0.1.0-rc.1 is the initial new release candidate version of this API, including initial documentation.**
```
(there is no implementation code, right?)

Author: Fixed on commit 4c0f60d
